README.md

```diff
@@ -272,21 +316,21 @@ callback let you add assertions and enrich the test.
 
 * **_AFT.testVerbResponseEqualsError(testName, api, verb, args, expectedResponse)**
 
-    The inverse than above
+    The inverse than above.
 
 * **_AFT.testVerbCb(testName, api, verb, args, expectedResponse, callback)**
 
-    Test that the call of a verb with a custom callback. From this callback you
+    Test the call of a verb with a custom callback. From this callback you
     will need to make some assertions on what you need (verb JSON return object
     content mainly).
 
-    If you doesn't need to test the response simply specify an empty LUA table.
+    If you don't need to test the response simply specify an empty LUA table.
 
 * **_AFT.testVerbCbError(testName, api, verb, args, expectedResponse, callback)**
 
     Should return success on failure.
 
-* **_AFT.testEvtReceived(testName, eventName, timeout)**
+* **_AFT.testEvtReceived(testName, eventName)**
```

Comment:
Add a timeout event as last arguments even if not implemented right now, it shouldn't be long. So for one time we have the documentation more advanced than the code...